tests/zfs-tests/include/libtest.shlib

```diff
@@ -2843,3 +2843,30 @@ function block_device_wait
 		$UDEVADM settle
 	fi
 }
+
+function kill_process
+{
+	typeset -l i killpid=$1
+
+	#attempt to kill
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_must $KILL $killpid
+	fi
+	#verify killed worked
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_mustnot $KILL $killpid
+	fi
+}
+
+function start_io
+{
+	typeset -l i sizetouse=$1
+	typeset -l i blocksize=$2
+	typeset -l i filename=$3
+
+	#start some IO
+	#This used to be: $FILE_TRUNC -f $((64 * 1024 * 1024)) -b 8192 -c 0 -r $TESTDIR/$TESTFILE1 &
+	dd_string="$DD if=/dev/urandom iflag=fullblock bs=$blocksize count=$sizetouse of=$filename & "
+	log_note "$dd_string"
+	dd_string
```

Comment:
The fact that `start_io` will start a background process isn't at all clear from the function name.  If we're going to add helpers for this kind of thing let's make it clear this is async.  Call it say `start_io_async`.

Why exactly can't FILE_TRUNC be used for this?  It would be preferable to using `/dev/urandom` which is meant for small chunks of random data and will be exceptionally slow compared to the alternatives.

Why not use `log_must` for this dd for logging purposes?